### ISSUE TITLE
Skip CI based on commit message

### ISSUE THREAD

**Issue by tjtelan** [shankj3/ocelot#42]
For example, we don't need to rebuild when config files are changed. It would be nice if we could proactively skip a build.

**Comment by tjtelan** [comment]
Identified an edge case where java builds in feature branches change their artifact name to remove *-SNAPSHOT, and it causes builds to fail (bc the build is on a non-master branch). It would be practical for devs to be able to skip in this situation.

However, it leaves a case where the -SNAPSHOT will need to be reintroduced in the branch if any new commits are being made after a merge into master... But, this is pretty buildtool specific, and might be on the devs to define best practices.

**Comment by shankj3** [comment]
like this? 

```yaml
stages:
  - name: conditional event
    trigger:
      branches:
        - release\/.*
    skip: 
      commitText: [nobuildmeplz]
```

also is there any other conditional we'd want for a skip action? @tjtelan

**Comment by shankj3** [comment]
closed by #60
